runtime/src/iree/hal/drivers/cuda/BUILD.bazel

```diff
@@ -73,6 +73,8 @@ iree_runtime_cc_library(
     srcs = [
         "cuda_headers.h",
         "dynamic_symbols.c",
+        "status_util.c",
+        "status_util.h",
```

Comment:
you don't want to include this in both this library and the other - you can put status_util in its own file or export the header here for use by the main cuda rule (so srcs here would have status_util.c, hdrs would have status_util.h, and the :cuda rule would lose both)

Reply:
Good point. Thanks